providers/redfish/firmware.go

```diff
@@ -106,7 +106,23 @@ func (c *Conn) FirmwareInstall(ctx context.Context, component, applyAt string, f
 		"UpdateFile":       reader,
 	}
 
-	resp, err := c.runRequestWithMultipartPayload(http.MethodPost, "/redfish/v1/UpdateService/MultipartUpload", payload)
+	updateService, err := c.redfishwrapper.UpdateService()
```

Comment:
I suggest we update the `firmwareUpdateCompatible()` with the logic below, to return the update URI and it returns an error if the URI is not supported.